src/asm/section.c

```diff
@@ -416,6 +428,35 @@ void out_EndLoadSection(void)
 	loadOffset = 0;
 }
 
+void out_PushInlineFragmentSection(void)
+{
+	checkcodesection();
+
+	if (currentLoadSection)
+		fatalerror("`LOAD` blocks cannot contain inline fragments\n");
```

Comment:
Why?

Reply:
It wouldn't be clear whether the inline fragment counts as part of the RAM space or not.

Reply:
I see. We can always add support later on if the need arises.